Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:12
+
+WORKDIR /usr/src/app
+
+COPY package.json yarn.lock ./
+
+RUN yarn install --prod --ignore-scripts
+
+COPY . .
+
+EXPOSE 8080
+
+CMD [ "node", "app.js" ]
```

Comment:
`"yarn", "server"` ?

Reply:
maybe can't ? no yarn command in Image? 
`"npm", "run", "server"` ?